isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -424,10 +472,10 @@ one time) may be useful depending on the complexity of your processing requireme
 
   <bibliography>
      <title>Installing ISIS</title>
-    <brief>Downloading and installing ISIS 3</brief>
+    <brief>Downloading and installing ISIS3</brief>
     <description>
       This document describes how to download/compile and install a
-      binary version of ISIS 3.
+      binary version of ISIS3.
     </description>
     <author>Kristin Berry</author>
```

Comment:
I think that it is awesome that @kberryUSGS took the lead and created (and/or modified) all of this. For the ISIS3 devs., does it make sense that this is by the ISIS3 development team?

Just today a whole bunch of folks (@twilson271828 @AstroKEW78 @thareUSGS @jessemapel - I am missing more I am sure) have contributed to this.

Reply:
@twilson271828 Has been taking the lead on updating this. This is an old document from before cmake. We should probably cite Tyler or the Astrogeology Science Center Software Developers.

Reply:
Apologies @twilson271828. Thanks for taking the lead on this. If okay with you, how about ASC Software Developers? Just because of the team effort that you have been coordinating on this.

Reply:
I'm okay with that.